sway/desktop/output.c

```diff
@@ -915,6 +1001,7 @@ static void send_frame_done(struct sway_output *output, struct timespec *when) {
 
 	struct sway_container *workspace = output_get_active_workspace(output);
 	send_frame_done_container(&data, workspace);
+	send_frame_done_container(&data, workspace->sway_workspace->floating);
```

Comment:
`output_view_for_each_surface` has not been updated for floating containers, so this can't work

Reply:
I don't quite understand this comment. Is this still relevant after my most recent commits, and if so can you please clarify what you mean?